src/connection.ts

```diff
@@ -1115,6 +1136,15 @@ export class Connection extends EventEmitter {
           stream._executeHold(requestPacket.sequence.toString())
         }
 
+        for (let frame of responsePacket.frames) {
+          if (frame.type === FrameType.StreamReceipt) {
+            const stream = this.streams.get(frame.streamId.toNumber())
+            if (stream) {
+              stream.emit('receipt', frame.receipt)
```

Comment:
Should receipts instead be emitted in the `outgoing_money` event so that sender's can more easily verify receipt amounts?